components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/import-file-dialog.js

```diff
@@ -16,266 +16,193 @@
-                    var payload = "configName=" + btoa("workspace" + app
-                        .getPathSeperator() + options.configName);
+                    function alertError(errorMessage) {
+                        var errorNotification = getErrorNotification(errorMessage);
+                        $(notification_container).append(errorNotification);
+                        errorNotification.fadeTo(2000, 200).slideUp(1000, function () {
+                            errorNotification.slideUp(1000);
+                        });
+                    }
 
-                    $.ajax({
-                        type: "POST",
-                        contentType: "text/plain; charset=utf-8",
-                        url: saveServiceURL,
-                        data: payload,
-                        async: false,
-                        success: function (response) {
-                            data = response;
-                        },
-                        error: function(xhr, textStatus, errorThrown){
-                            data = client.getErrorFromResponse(xhr, textStatus, errorThrown);
-                            log.error(data.message);
+                    function getErrorNotification(detailedErrorMsg) {
+                        var errorMsg = "Error while importing file";
+                        if (!_.isEmpty(detailedErrorMsg)) {
+                            errorMsg = (detailedErrorMsg);
                         }
-                    });
-                    return data;
-                }
-
-                function importConfiguration() {
-                    var defaultView = {configLocation: selectedFilePath};
-                    var workspaceServiceURL = app.config.services.workspace.endpoint;
-                    var importServiceURL = workspaceServiceURL + "/import";
-
-                    var path = defaultView.configLocation;
-                    $.ajax({
-                        url: importServiceURL,
-                        type: "POST",
-                        data: path,
-                        contentType: "text/plain; charset=utf-8",
-                        async: false,
-                        success: function (data, textStatus, xhr) {
-                            if (xhr.status == 200) {
-                                var pathArray = _.split(path, self.app.getPathSeperator()),
-                                    fileName = _.last(pathArray),
-                                    folderPath = _.join(_.take(pathArray, pathArray.length -1),
-                                        self.app.getPathSeperator());
+                        return $(
+                            "<div style='z-index: 9999;' style='line-height: 20%;' class='alert alert-danger'" +
+                            " id='error-alert'>" + "<span class='notification'>" +
+                            errorMsg +
+                            "</span>" +
+                            "</div>");
+                    }
```

Comment:
Shall we format this to align according to elements

Reply:
Fixed with #1690